### PR TITLE
 [Keyboard] Added nibell micropad4x4

### PR DESCRIPTION
**Pull request by Nibell**
## Description

Add nibell/micropad4x4

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nibell/micropad4x4/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    * ├─────┼─────┼─────┼─────┤
+    * │     │     │     │     │
+    * └─────┴─────┴─────┴─────┘
+     */
+    [3] = LAYOUT_ortho_4x4(
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
```
Comment (drashna) [inline keyboards/nibell/micropad4x4/keymaps/default/keymap.c:89]: Empty layers should be deleted.  All they do is take up firmware space. 
```suggestion
```

File: keyboards/nibell/micropad4x4/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+* Keyboard Maintainer: [nibell](https://github.com/Nibell)
+Hardware Supported: Raspberry Pi Pico 
+* Hardware Availability: [GitHub](https://github.com/Nibell/Keyboards/tree/main/micropad4x4)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nibell/micropad4x4:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (drashna) [inline keyboards/nibell/micropad4x4/readme.md:15]: No way to enter the bootloader?  
Otherwise, bootloader section should be re-added

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/nibell/micropad4x4/readme.md (hunk 2 above)
Comment (Nibell) [reply keyboards/nibell/micropad4x4/readme.md:22]: Added bootloaderkey in layer and update docs

File: keyboards/nibell/micropad4x4/keymaps/default/keymap.c (hunk 1 above)
Comment (Nibell) [reply keyboards/nibell/micropad4x4/keymaps/default/keymap.c]: Updated layers

**Review by drashna — APPROVED** [review]

File: keyboards/nibell/micropad4x4/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    * │   1  │  2  │  3   │   4   │
+    * ├──────┼─────┼──────┼───────┤
+    * │      │     │RSHIFT│ RCTRL │
+    * └──────┴─────┴──────┴───────┘
+     */
+    [0] = LAYOUT_ortho_4x4(
+        KC_NUM, KC_NO,  KC_NO,          QK_BOOT,
+        TO(1),  TO(2),  TO(3),          KC_NO,
+        KC_1,   KC_2,   KC_3,           KC_4,
+        KC_NO,  KC_NO,  KC_RIGHT_SHIFT, KC_RIGHT_CTRL
```
Comment (fauxpark) [inline keyboards/nibell/micropad4x4/keymaps/default/keymap.c:34]:
```suggestion
        KC_NO,  KC_NO,  KC_RSFT, KC_RCTL
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/nibell/micropad4x4/keymaps/default/keymap.c (hunk 3 above)
Comment (Nibell) [reply keyboards/nibell/micropad4x4/keymaps/default/keymap.c]: changed :)

File: keyboards/nibell/micropad4x4/keyboard.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        "cols": ["GP2", "GP3", "GP4", "GP5"],
+        "rows": ["GP10", "GP11", "GP12", "GP13"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xFEED",
+        "vid": "0x4C4E"
+    },
```
Comment (dunk2k) [inline keyboards/nibell/micropad4x4/keyboard.json:23]:
```suggestion
    },
    "community_layouts": ["ortho_4x4"],
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
community layout support

File: keyboards/nibell/micropad4x4/readme.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+## Bootloader
+
+Enter the bootloader in 2 ways:
+
+* **Physical reset button**: Hold the button down on the raspberry pi pico while plugging it in. There is no reset button on the pcb.
+* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nibell/micropad4x4:default
```
Comment (waffle87) [inline keyboards/nibell/micropad4x4/readme.md:20]:
```suggestion
    make nibell/micropad4x4:default
    
Flashing example for this keyboard:

    make nibell/macropad4x4:default
```

**Review by waffle87 — APPROVED** [review]

**Pull request merged**